mne/beamformer/_lcmv.py

```diff
@@ -398,6 +398,48 @@ def apply_lcmv_raw(raw, filters, start=None, stop=None, max_ori_out='signed',
     return next(stc)
 
 
+@verbose
+def apply_lcmv_cov(data_cov, filters, verbose=None):
+    """Apply Linearly Constrained  Minimum Variance (LCMV) beamformer weights.
+
+    Apply Linearly Constrained Minimum Variance (LCMV) beamformer weights
+    to a data covariance matrix to estimate source power.
+
+    Parameters
+    ----------
+    data_cov : instance of Covariance.
```

Comment:
No period at the end, this is probably messing up the linking

https://12944-1301584-gh.circle-artifacts.com/0/dev/generated/mne.beamformer.apply_lcmv_cov.html#mne.beamformer.apply_lcmv_cov